src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -87,7 +87,7 @@ export class Main extends React.Component {
   }
 
   bindModalTriggers = () => {
-    const triggers = document.querySelectorAll('.signin-signup-modal-trigger');
+    const triggers = Array.from(document.querySelectorAll('.signin-signup-modal-trigger'));
```

Comment:
I remember the days of using `[].forEach.call` in order to iterate a Node list

Reply:
I still use `[].forEach.call` if I'm in a hurry.